### ISSUE TITLE
August 2021 Community Meeting tweet request (Monday release)

### ISSUE THREAD

**Issue by nimbinatus** [kubernetes-sigs/contributor-tweets#57]
**Provide the actual tweet content between the commented colons**

Join us for the monthly #Kubernetes Community Meeting, the place for #k8s contributors to discuss current activities happening across the ecosystem. This month, SIG Release will be holding the retro for 1.22. Find more information on the mailing list or at https://www.kubernetes.dev/events/community-meeting/

**Comment by chris-short** [comment]
@nimbinatus Did this go out as intended?

**Comment by nimbinatus** [comment]
I don't think so. I don't remember seeing it. I don't believe the email went out either, though.
